Sources/LLBuildManifest/ManifestWriter.swift

```diff
@@ -179,3 +216,74 @@ extension Node {
         }
     }
 }
+
+extension String {
+    fileprivate var asJSON: String {
+        "\"\(self.jsonEscaped)\""
+    }
+
+    private var jsonEscaped: String {
```

Comment:
Directly lifted from TSC, suitable just for this use case. When we fully deprecate the `JSON` type, we can deprecate this in TSC too.

Reply:
is there nothing in Foundation that we can use instead of inventing our own? further why not use Codable?

Reply:
Nothing that they publicly expose. The new Swift Foundation has an internal `serializedForJSON` method on `String`, I'm not sure it's worth vendoring that instead of the TSC version? Additionally, we'd have to write our own encoder, since this is strictly neither JSON, nor YAML, it only uses JSON-escaping for strings. If we use `JSONEncoder` for some bits, without refactoring this much more we'd have to instantiate `JSONEncoder` on every call property call, which would be quite expensive and probably a significant slowdown for large projects.

Reply:
Hm, my bad, my impression was that it was somewhat custom, but I see it does mention YAML. Would you be opposed just refactoring this to render the build manifest with `Codable` and `JSONEncoder` in a future PR?

Reply:
Since we don't need to parse these manifest, I think rendering to JSON would mean we don't have to pull in additional dependencies for YAML support.